src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.model;
+
+import static core.basesyntax.model.FruitTransaction.Operation.convertor;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+
+    @Test
+    void convert_b_ToEnumBalance_shouldReturnBalance() {
```

Comment:
use the same method pattern
```suggestion
    void convert_balanceOperationCode_shouldReturnBalance() {
```